hub/apps/windows-app-sdk/notifications/app-notifications/app-notifications-quickstart.md

```diff
@@ -49,7 +49,8 @@ If your app is unpackaged (that is, it lacks package identity at runtime), then
 
 If your app is packaged (including packaged with external location):
 
-1. Open your **Package.appxmanifest**. 
+1. Open your **Package.appxmanifest**.
+1. Add `xmlns:com="http://schemas.microsoft.com/appx/manifest/com/windows10"` and `xmlns:desktop="http://schemas.microsoft.com/appx/manifest/desktop/windows10"` namespaces to `<Package>`

```

Comment:
We don't  have a fully packaged sample for AppNotifications yet but in a prototype I've been toying around with, I have the following entires in the appxmanifest:
-  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
-  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
-  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"

Not sure if they are needed or not, just puting this here in case.